script.js

```diff
@@ -0,0 +1,26 @@
+function closeMenu() {
+  document.querySelector('.menu-bars').classList.remove('hide');
+  document.querySelector('#desktop-nav').classList.remove('show');
+  document.querySelector('#desktop-nav ul').classList.remove('show');
+  document.querySelector('main').classList.remove('hide');
+  document.querySelector('footer').classList.remove('hide');
+  document.querySelector('h1').classList.remove('hide');
+  document.querySelector('header').classList.remove('menu-visible');
+  document.querySelector('.close-btn').classList.add('hide');
+}
```

Comment:
- [ ] I would strongly recommend you add functionality for the mobile menu such that when the user clicks (or taps) any option of the mobile menu, the mobile menu disappears just the way the menu disappears when you click the hamburger. This is equally a good user experience.